From 53eddceef2eaa9adda3a3f8c90ceae7c4870dc7c Mon Sep 17 00:00:00 2001
From: Shunqing Chen <csq@rock-chips.com>
Date: Wed, 16 Dec 2020 03:23:27 +0000
Subject: [PATCH] arm64: configs: rockchip_defconfig: enable sii902x

Add RGB/BT1120 to HDMI support.
This is needed by rk3568-evb2-lp4x-v10-bt1120-to-hdmi.

Signed-off-by: Shunqing Chen <csq@rock-chips.com>
Change-Id: I40987ec96fee9d4d7f99970222bae487f58927df
---
 arch/arm64/configs/rockchip_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/rockchip_defconfig b/arch/arm64/configs/rockchip_defconfig
index c9a1e652cda1..a6d4119983e1 100644
--- a/arch/arm64/configs/rockchip_defconfig
+++ b/arch/arm64/configs/rockchip_defconfig
@@ -594,6 +594,7 @@ CONFIG_DRM_ROCKCHIP_RK628=y
 CONFIG_DRM_PANEL_SIMPLE=y
 CONFIG_DRM_RK630_TVE=y
 CONFIG_DRM_RK1000_TVE=y
+CONFIG_DRM_SII902X=y
 CONFIG_DRM_DW_HDMI_I2S_AUDIO=y
 CONFIG_DRM_DW_HDMI_CEC=y
 CONFIG_MALI400=y
-- 
2.35.3

